Demo/Application/Base/SceneDelegate.swift

```diff
@@ -27,9 +27,9 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
     func scene(_ scene: UIScene, continue userActivity: NSUserActivity) {
-        if let returnURL = userActivity.webpageURL {
-            // TODO: implementation - pass full URL to BT SDK
+        if let returnURL = userActivity.webpageURL, returnURL.path == "/braintree-payments" {
             print("Returned to Demo app via universal link: \(returnURL)")
+            _ = BTAppContextSwitcher.sharedInstance.handleOpen(returnURL)
```

Comment:
Wondering to know if we can use `@discardableResult`

Reply:
🤔 I am down - historically we haven't used it but I just tested and it doesn't seem to be a breaking change for either Obj-C or Swift interface, so no reason not to. Also tested to ensure that merchants that may _want_ to capture the result will still be able to. As long as CI is happy lets go for it! Updated here: https://github.com/braintree/braintree_ios/pull/1214/commits/c0a8e49f6d968aff9951ad18bcbb08771d3b66e4